src/sec_limit_continuity.mbx

```diff
@@ -26,8 +27,12 @@
           <me>
             \displaystyle \lim_{x\rightarrow c^-} f(x) = L,
           </me>
-          means that given any <m>\varepsilon > 0</m>, there exists <m>\delta > 0</m> such that for all <m>x\lt  c</m>,
-          if <m>\abs{x - c} \lt  \delta</m>, then <m>\abs{f(x) - L} \lt  \varepsilon</m>.</p>
+          means that for <m>x \lt c</m>, given any <m>\varepsilon \gt 0</m>, there exists <m>\delta \gt 0</m> such that <!--for all <m>x\lt  c</m>,
+          if <m>\abs{x - c} \lt  \delta</m>, then <m>\abs{f(x) - L} \lt  \varepsilon</m>.--> 
+          if <m>x \lt  c-\delta</m>, then <m>\abs{f(x) - L} \lt  \varepsilon</m>.
+        </p>
+          <aside><p>In <xref ref="def_onesidedlimit">Definition</xref>, saying <m>x \lt  c-\delta</m> is equivalent to saying that <m>x</m> is at most <m>\delta</m> units to the <em>left</em> of <m>c</m>. We are approacing <m>c</m> from the left, so we are only concerned with being near <m>c</m> from the left side. </p> </aside>
```

Comment:
I'm not sure it is a good thing to simplify |x - c| to c - x. The theorem is still valid with |x - c| and leaving it that way makes it easier to remember. (Since it is the same statement as for a limit in general, just with the initial restriction on where x comes from). I'd rather be a student remembering just |x - c| than remembering |x - c| in some situation, c - x, in others, and x - c in others. It would be easy to mix up the last two.